models/release.go

```diff
@@ -395,7 +395,7 @@ func UpdateRelease(doer *User, gitRepo *git.Repository, rel *Release, attachment
 		return err
 	}
 
-	err = rel.loadAttributes(x)
+	err = rel.loadAttributes()
```

Comment:
I think we should keep the engine here.

Reply:
But it was never used inside of `loadAttributes`?